liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -597,12 +602,17 @@ protected Object readDefaultValue(CachedRow columnMetadataResultSet, Column colu
 
     private void readDefaultValueForMysqlDatabase(CachedRow columnMetadataResultSet, Column column, Database database) {
         try {
+            List<String> parameters = new ArrayList<>(3);
+            parameters.add(column.getSchema().getName());
+            parameters.add(column.getRelation().getName());
+            parameters.add(column.getName());
+            StringBuilder selectQuery = new StringBuilder("SELECT EXTRA FROM INFORMATION_SCHEMA.COLUMNS\n")
+                    .append("WHERE TABLE_SCHEMA = '?'\n")
```

Comment:
Hey, checking again: is it working ? we are adding quotes around the question mark, I guess it should fail...

Reply:
I just removed these singles quotes we forgot to remove. 

Thanks,
Daniel.